data/presets/natural/tree.json

```diff
@@ -1,14 +1,20 @@
 {
     "icon": "maki-park",
     "fields": [
+        "genus",
+        "species",
+        "taxon",
         "leaf_type_singular",
         "leaf_cycle_singular",
-        "denotation",
-        "height",
-        "diameter"
+        "start_date"
```

Comment:
I would move this to the `moreFields` section: If there is no dedicated signage at the tree, this information is almost impossible to deduce on the ground. Imports and OGD data might make it possible to map this information, but for that use cases it is sufficient to have this field only show if the information is already present.

Reply:
Done 👍 Thanks @tyrasd